Documentation/blame-options.txt

```diff
@@ -41,7 +41,7 @@ include::line-range-format.txt[]
 	Show in a format designed for machine consumption.
 
 --line-porcelain::
-	Show the porcelain format, but output commit information for
+	Show the porcelain format, but the output commit information for
```

Comment:
I believe this is incorrect. "Show the porcelain format" is the imperative form, and "output commit information" is *also* the imperative form. If you insert "the", you turn "output" into a noun, and the sentence is missing a verb.

Do you agree? If yes, please abandon this PR ;-)

Reply:
Savage :D